src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents contact information of Person
+ * Super-class of Email, Phone, Address
```

Comment:
There is no need to mention the super-class thing and where the class is used in the header comment.